multi-agent/lib/mas/resources/builtin_service.py

```
@@ -0,0 +1,583 @@
+            resource.category, resource.type
+        )
+        secret_configurable_keys = configurable_keys & sensitive_keys
+        if not secret_configurable_keys:
+            return config
+        stripped = dict(config)
+        for key in secret_configurable_keys:
+            stripped.pop(key, None)
+        return stripped
+
+    def find_missing_required_overlay_fields(
+        self, resource: Resource, resolved_config: Any,
```

Ruff (0.15.21)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `resolved_config`

(ANN401)